src/command_modules/azure-cli-network/azure/cli/command_modules/network/_validators.py

```diff
@@ -150,6 +150,19 @@ def validate_nsg_name_or_id(namespace):
                 type='networkSecurityGroups',
                 name=namespace.network_security_group)
 
+def validate_peering_type(namespace):
+    if namespace.peering_type and namespace.peering_type.lower() == 'microsoftpeering':
```

Comment:
Yes, you are correct. enum_choice_list will actually force it to MicrosoftPeering so I'll update that.